File: src/main/java/com/gitlabci/plugin/language/GitlabYamlLanguage.java
package com.gitlabci.plugin.language;

import com.intellij.lang.Language;

public class GitlabYamlLanguage extends Language {

    public static final GitlabYamlLanguage INSTANCE = new GitlabYamlLanguage();

    private GitlabYamlLanguage() {
        super("GitlabYaml");
    }
}

File: src/main/kotlin/com/github/fiskirton/gitlabciplugin/language/GitlabYamlLanguage.kt
package com.github.fiskirton.gitlabciplugin.language

import com.intellij.lang.Language

class GitlabYamlLanguage private constructor() : Language("GitlabYaml") {

    companion object {
        val INSTANCE: GitlabYamlLanguage = GitlabYamlLanguage()
    }}